frontend/src/modelTodo.ts

```diff
@@ -0,0 +1,11 @@
+export interface Todo {
+    id? : string;
```

Comment:
warum das '?' - muss id optional sein?

Reply:
André meint, falls es kein id kommt, es könnt optinial in meinem interface sein. Es wird sowie in backend generiert

Reply:
Ok, passt schon :)